src/app/submission/Submission.ts

```diff
@@ -116,12 +116,18 @@ export class Submission {
       console.error(error);
     }
 
+    const timestamp = this.parsedSubmission.metadata.timestamp.slice(0, -1) as [number, number, number, number, number, number];
+    timestamp.slice(0);
```

Comment:
Comment er bij met waarom we dit weg gooien?

Reply:
Goed punt, gedaan!